apcd-cms/src/apps/admin_regis_table/templates/create_submitter_modal.html

```diff
@@ -0,0 +1,70 @@
+<div id="viewSubmitterModal_{{r.reg_id}}" class="modal fade" role="dialog">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header">
+              <h4 class="modal-title">Create New Submitter</h4>
+              <button type="button" class="close" data-dismiss="modal">
+                <span aria-hidden="true">&#xe912;</span>
+              </button>
+            </div>
+            <div class="modal-body" style="height: auto;">
+                
+                
+                  <h4>Organization</h4>
+                  
+                  <dd class="c-data-list__value">{{r.view_modal_content.type}}</dd>                       
+                                
+                  <dd class="c-data-list__value">{{r.view_modal_content.biz_name}}</dd>
+                                
+                  <dd class="c-data-list__value">{{r.view_modal_content.address}}</dd>
+                  <dd class="c-data-list__value">{{r.view_modal_content.city}}, {{r.view_modal_content.state}}</dd>
+                  <dd class="c-data-list__value">{{r.view_modal_content.zip}}</dd>
+                
+                  <dd class="c-data-list__value">{{r.view_modal_content.reg_status}}</dd>
```

Comment:
**Concern.** This is empty when I open the modal. Is it cruft or a bug?

Reply:
All of these fields are empty? I saw the registration status was empty, but everything else shown here looked like it was populated.

Reply:
Okay, I'll look again at `{{r.view_modal_content.reg_status}}`.